jellyfin_kodi/jellyfin/ws_client.py

```diff
@@ -79,21 +79,23 @@ def on_error(self, ws, error):
 
     def on_open(self, ws):
         LOG.info("--->[ websocket ]")
-        self.client.jellyfin.post_capabilities({
-            'PlayableMediaTypes': "Audio,Video",
-            'SupportsMediaControl': True,
-            'SupportedCommands': (
-                "MoveUp,MoveDown,MoveLeft,MoveRight,Select,"
-                "Back,ToggleContextMenu,ToggleFullscreen,ToggleOsdMenu,"
-                "GoHome,PageUp,NextLetter,GoToSearch,"
-                "GoToSettings,PageDown,PreviousLetter,TakeScreenshot,"
-                "VolumeUp,VolumeDown,ToggleMute,SendString,DisplayMessage,"
-                "SetAudioStreamIndex,SetSubtitleStreamIndex,"
-                "SetRepeatMode,"
-                "Mute,Unmute,SetVolume,"
-                "Play,Playstate,PlayNext,PlayMediaSource"
-            ),
-        })
+        if settings('remoteControl.bool'):
+            self.client.jellyfin.post_capabilities({
+                'PlayableMediaTypes': "Audio,Video",
+                'SupportsMediaControl': True,
```

Comment:
Probably want to still send this, with `SupportsMediaControl` False and empty `SupportedCommands`?

Or is capabilities exclusively used for remote control and not media?

After this we'd still accept and act on remote control commands, if the server is made to ignore the data sent here in some way.

Reply:
I tried just sending `{'SupportsMediaControl': False}` and the server laughed at me and kept allowing remote control anyway.  I can't say for certain if that's the actual results or if it was just a quirk of my kodi instance crashing every 30 seconds though.  Sending an empty payload was the only reliable way I had it stop showing up in the play to menu.

Reply:
Well, if playback still works with this, then it's fine.

Reply:
I can't test any of it.  If you want to test flipping the payload around and seeing if there's any difference then cool, but I'm dev-ing blind here until I figure out what's causing this addon to hard crash Kodi on my workstation.